src/webview/MessageListWeb.js

```diff
@@ -12,18 +12,42 @@ import { getInputMessages } from './webViewHandleUpdates';
 import * as webViewEventHandlers from './webViewEventHandlers';
 import { base64Utf8Encode } from '../utils/encoding';
 
-export default class MessageListWeb extends Component<Props> {
+type State = {
+  isWebViewReady: boolean,
+};
+
+export default class MessageListWeb extends Component<Props, State> {
   context: Context;
   props: Props;
   webview: ?Object;
-  isReady: boolean;
   unsentMessages: WebviewInputMessage[] = [];
 
+  state = {
+    isWebViewReady: false,
+  };
+
   static contextTypes = {
     styles: () => null,
     theme: () => null,
   };
 
+  /*
+  *
+  * Synchronizing setup with webview
+  *
+  */
+  sendTestPing = (): void => {
+    if (this.state.isWebViewReady) {
+      return;
+    }
+    this.sendMessages([{ type: 'ping' }]);
```

Comment:
I would have called this event 'Hello'.
This is way more typical greeting when doing a handshake, IRL and digitally.
`ping` tends to mean 'are you still alive'.